graphics/blender/Portfile

```diff
@@ -78,6 +78,29 @@ depends_lib-append  port:python${py_ver_nodot} \
                     port:lzo2 \
                     port:gflags
 
+# Blender does not depend directly on OpenSSL, but it is brought in by
+# many of its dependencies.
+#
+# * OpenSSL's libraries are never linked to in Blender.
+# * OpenSSL's header files are never included.
+# * The OpenSSL part of Blender's dependencies, such as ffmpeg and
+#   OpenImageIO, are never used. In fact, if we were to leave Blender
+#   unpatched and let Blender compile ffmpeg and OpenImageIO internally
+#   as statically-linked libraries, they would be getting compiled using
+#   flags such as -DUSE_OPENSSL=OFF and --disable-openssl.
+#
+# Reference: See portfile for Octave.
+license_noconflict-append openssl ffmpeg openimageio
```

Comment:
Openssl shouldn't be in this list if this port doesn't use it. List the things you do use that shouldn't be considered conflicting.

Reply:
I got this directly from the Octave portfile. Octave doesn't depend on openssl either, yet they add openssl to their list... I suspect in order to resolve this conflict message on the builders:

"blender" is not distributable because its license "gpl" conflicts with license "OpenSSL" of dependency "openssl"

I can certainly remove it, if you think it won't get flagged as a conflict.

Reply:
If you list all of the direct dependencies that have openssl (or other apparently-conflicting ports) in their recursive dependencies then the port will be considered distributable.